### DISCUSSION TITLE
[Enhancement] Custom Layout for CollectionView

### DISCUSSION THREAD

**Discussion by artemvalieiev** [dotnet/maui#9028] [Ideas]
Hello,
I've ported [ChatLayout](https://github.com/ekazaev/ChatLayout) and want to use it in the upcoming MAUI app, is there support to substitute default collectionviewflowlayout on ios to use my collection view layout, or maybe there is a mechanism to write custom layouts without implementing native layouts?

Thanks.

**Comment by SarthakGz** [comment]
Everything is possible if its available on iOS native, though it would be very time taking process due to lack of references in MAUI/Xamarin and smaller community. You have to take the references from Native and convert them to C# and read MAUI repo code for better understanding
